cloud/deployment/environment.sh

```diff
@@ -0,0 +1,12 @@
+
+export PROJECT_ID=arcs-project-223901
```

Comment:
Consider making these default values / allow developers to set their own env variables.

https://github.com/dylanaraps/pure-sh-bible#default-value

Reply:
Added TODO(cromwellian) @cromwellian